spec/features/applicant_page_spec.rb

```diff
@@ -80,6 +80,39 @@
     end
   end
 
+  context "when employed but also in receipt of a passporting benefit" do
+    let(:estimate_id) { SecureRandom.uuid }
+    let(:mock_connection) do
+      instance_double(CfeConnection,
+                      create_assessment_id: estimate_id,
+                      create_applicant: nil,
+                      create_proceeding_type: nil,
+                      api_result: calculation_result)
+    end
+    let(:calculation_result) { CalculationResult.new FactoryBot.build(:api_result) }
+
+    before do
+      allow(CfeConnection).to receive(:connection).and_return(mock_connection)
+      visit_applicant_page
+
+      select_radio_value("applicant-form", "proceeding-type", "se003") # non-domestic abuse case
+      select_applicant_boolean(:over_60, false)
+      select_applicant_boolean(:employed, true)
+      select_applicant_boolean(:passporting, true)
+
+      click_on "Save and continue"
+      select_radio_value("property-form", "property-owned", "none")
+      click_on "Save and continue"
+      select_boolean_value("vehicle-form", :vehicle_owned, false)
+      click_on "Save and continue"
```

Comment:
I think there might be some methods in form_helpers to shorten this a little bit